chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertifiedProductDetailsManager.java

```diff
@@ -719,6 +720,14 @@ private CertifiedProductSearchDetails getCertifiedProductSearchDetails(Certified
         searchDetails.setQmsStandards(getCertifiedProductQmsStandards(dto.getId()));
         searchDetails.setTargetedUsers(getCertifiedProductTargetedUsers(dto.getId()));
         searchDetails.setAccessibilityStandards(getCertifiedProductAccessibilityStandards(dto.getId()));
+        searchDetails.setRwtPlansUrl(dto.getRwtPlansUrl());
+        if (Objects.nonNull(dto.getRwtPlansCheckDate())) {
```

Comment:
Why does this need to check against null where the URL doesn't, and it doesn't look like any of the other fields do either?